jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -130,6 +130,11 @@ public String getJarPluginName() {
     return JAR_PLUGIN_NAME;
   }
 
+  @Override
+  public boolean isWarProject() {
+    return false; // TODO: to be implemented. For now, assume false.
```

Comment:
Is this for a future PR?

Reply:
Had to add the method, since we should implement the interface.